crates/primitives/src/receipt.rs

```diff
@@ -21,13 +25,10 @@ pub struct Receipt {
     /// Gas used
     pub cumulative_gas_used: u64,
     /// Log send from contracts.
-    #[cfg_attr(
-        any(test, feature = "arbitrary"),
-        proptest(
-            strategy = "proptest::collection::vec(proptest::arbitrary::any::<Log>(), 0..=20)"
-        )
-    )]
```

Comment:
we removed arbitrary support here because the implementation is handrolled below, correct?

Reply:
Yes, but will page @clabby since he authored the change iirc

Reply:
Yes, removed to hand-roll below - non-deposit tx type receipts should never have a deposit nonce.